contracts/Vault.vy

```diff
@@ -1155,7 +1157,7 @@ def expectedReturn(_strategy: address = msg.sender) -> uint256:
 
 
 @external
-def report(_return: uint256) -> uint256:
+def report(_gain: uint256, _loss: uint256) -> uint256:
```

Comment:
is it a wash if gain and loss are reported by any chance with same number?

Reply:
A reported loss gets accounted for first, reducing debt, so it'll look like you earned more with less I suppose.

Reply:
makes sense, would it be good to add a test case to check values after side effects?

Reply:
We added a test case in 1cb83d1, do you think it can be improved?

Reply:
yeah, maybe we can add a mock strategy that sends bad signals in testing like gain=100K and loss=100K just to check the state of the vault @Macarse, would that make sense?

Reply:
Ill add it as an issue to track to improve testing, just for completion sake i would say, glad to take a stab at it